features/support/topic_helper.rb

```diff
@@ -52,9 +38,17 @@ def stub_topic_lookups
         ],
       },
       links: {
-        "parent" => [
-          "title" => "Oil and Gas",
-          "base_path" => "/oil-and-gas",
+        parent: [
+          { title: "Oil and Gas", base_path: "/oil-and-gas" }
+        ],
+        topic_content: [
```

Comment:
Yes, that works, thanks! I've updated the topic content item, but I haven't done it for mainstream browse page tests so I'll do that now.